include/deal.II/base/table.h

```diff
@@ -1903,7 +1903,7 @@ TableBase<N,T>::reinit (const TableIndices<N> &new_sizes,
   // if new size is nonzero: if necessary allocate additional memory, and if
   // not fast resize, zero out all values)
   values.resize_fast (new_size);
-  if (!fast)
+  if (!omit_zeroing_entries)
```

Comment:
That's what I did -- I'm now saying that if the parameter is `false`, we do set all elements to what you get from default initialization; otherwise, the memory remains uninitialized or otherwise undefined.